soroban-token-sdk/Cargo.toml

```diff
@@ -11,8 +11,10 @@ edition = "2021"
 rust-version.workspace = true
 
 [lib]
-crate-type = ["cdylib", "rlib"]
```

Comment:
The cdylib is a hold over from when we used to use this lib as a way to build a wasm to get the contract spec, which we don't do anymore. The 'rlib' is unnecessary.